examples/basic-tf-example/porter.yaml

```diff
@@ -50,21 +113,51 @@ show:
 plan:
   - terraform:
       description: "Invoke 'terraform plan'"
-      flags:
```

Comment:
I removed these because they aren't helpful in an example, we don't actually want to suppress plan output or turn off color.